ci/deps/azure-35-compat.yaml

```diff
@@ -26,5 +26,5 @@ dependencies:
   - pip
   - pip:
     # for python 3.5, pytest>=4.0.2 is not available in conda
-    - pytest>=4.0.2
+    - pytest>=4.0.2,!=4.6.0
```

Comment:
see what they used on the last good build and pin to that

Reply:
4.5.0